### PR TITLE
Make image override by each image

### PR DESCRIPTION
**Pull request by xrajesh**
## Description
To avoid the merge conflict from konflux updates in the same line , we are making the argument specific to each image .

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: bundle/manifests/lightspeed-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -372,15 +372,16 @@ spec:
                       - --metrics-bind-address=:8443
                       - --secure-metrics-server
                       - --cert-dir=/etc/tls/private
-                      - --images=lightspeed-service=registry.redhat.io/openshift-lightspeed-beta/lightspeed-service-api-rhel9@sha256:0a53308b64905ac444c224f4b5ad744b891bdd790a49367336ebe72e513dfdef,console-plugin=registry.redhat.io/openshift-lightspeed-beta/lightspeed-console-plugin-rhel9@sha256:5538f2b63951e4eca4e57284165be6b1c1ae5c522c624380512081ac999214be
+                      - --service-image=registry.redhat.io/openshift-lightspeed-beta/lightspeed-service-api-rhel9@sha256:0a53308b64905ac444c224f4b5ad744b891bdd790a49367336ebe72e513dfdef
+                      - --console-image=registry.redhat.io/openshift-lightspeed-beta/lightspeed-console-plugin-rhel9@sha256:5538f2b63951e4eca4e57284165be6b1c1ae5c522c624380512081ac999214be
```
Comment (syedriko) [inline bundle/manifests/lightspeed-operator.clusterserviceversion.yaml:376]: @xrajesh Would just breaking up this long line into two work?
```
      - |
          --images=lightspeed-service=registry.redhat.io/openshift-lightspeed-beta/lightspeed-service-api-rhel9@sha256:...,
          console-plugin=registry.redhat.io/openshift-lightspeed-beta/lightspeed-console-plugin-rhel9@sha256:...
```

Comment (xrajesh) [reply bundle/manifests/lightspeed-operator.clusterserviceversion.yaml:384]: @syedriko I wanted this to be clean , and wanted separate params.  PTAL

Comment (syedriko) [reply bundle/manifests/lightspeed-operator.clusterserviceversion.yaml:384]: By all means :)

**Comment by xrajesh** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / bundle"

**Pull request merged**